lms/static/styles/components/_StudentSelector.scss

```diff
@@ -1,29 +1,54 @@
 .StudentSelector {
   display: flex;
-  button {
-    background-color: transparent;
-    min-width: 80px;
-    border: none;
-    background-color: $grey-6;
-    transition: background-color 0.2s, opacity 0.2s;
-    outline: none;
-    &:hover {
-      cursor: pointer;
-      background-color: $grey-5;
-    }
-    &:disabled {
-      &:hover {
-        background-color: $grey-6;
-      }
-      cursor: default;
-      opacity: 0.5;
-    }
+}
+
+.StudentSelector-change-student {
+  svg {
+    color: $grey-5;
+  }
+
+  min-width: 60px;
+  height: 40px;
+  border: none;
+  background-color: $grey-1;
+  border: 1px solid $grey-3;
+  transition: background-color 0.2s, border-color 0.2s;
+  
+  &:focus {
+    outline: 1px solid $grey-5;
+    outline-offset: -1px;
+    outline-style: solid
+  }
+
+  &:hover {
+    cursor: pointer;
+    background-color: $grey-2;
+  }
+
+  &:disabled {
+    background-color: $grey-1;
+    cursor: default;
+    opacity: 0.5;
   }
 }
 
-.StudentsSelector__student {
-  padding: 10px 20px;
-  display: flex;
-  justify-content: space-around;
-  flex-flow: column;
+.StudentsSelector__students {
+  appearance: none;
+  border: 1px solid $grey-3;
+  border-left: none;
+  border-right: none;
+  border-radius: 0;
+  height: 40px;
+  padding: 5px 20px;
+
+  &:focus {
```

Comment:
Side note: It looks like some of this focus stuff may be generalize-able, a la the `outline-on-keyboard-focus` mixin in the client: https://github.com/hypothesis/client/blob/abaf9cf2070ceb4af00f54bd98280b9bebc3dc71/src/styles/mixins/focus.scss#L10

Reply:
we absolutely need a mixin for this, but i sorta wanted to wait until we started the pattern lib stuff and just doing dumb-simple-stupid for now 

thoughts :\

Reply:
This seems reasonable. Glad it's on your radar.